docs/integrations/delta.md

```diff
@@ -20,7 +20,17 @@ Because lakeFS is format-agnostic, you can save data in Delta format within a la
 
 {% include toc.html %}
 
-## Beta: Viewing Delta table changes in lakeFS 
+## Beta: Viewing Delta Lake table changes in lakeFS
+
+### Installing the Delta Lake diff plugin
+
+To enable the Delta Lake diff feature, you need to install a plugin. You will find the plugin binary in the
+[release](https://github.com/treeverse/lakeFS/releases/latest) tarball (versions >= 0.100.0).
+Put the `delta_diff` binary under `~/.lakefs/plugins/diff` on the machine where lakeFS is running. 
+
+**Notice**: If you're using the lakeFS [docker image](../quickstart/run.md#running-locally-with-docker), the plugin is installed by default.
+
+### Viewing Delta Lake table changes
 
 Using lakeFS you can compare different versions of Delta tables, view the table operations that have been done since the tables diverged, and the details of those operations.
```

Comment:
```suggestion
Using lakeFS you can

* compare different versions of Delta tables
* view the table operations that have been done since the tables diverged and see the details of those operations.
```

Reply:
I would also crop the two subsequent screenshots e.g. 
![CleanShot 2023-03-21 at 10 04 25](https://user-images.githubusercontent.com/3671582/226574470-325db0c4-789e-496c-992c-a2002655e801.png)

I'd also move `Expanding the delete operation, we learn that all movies with a rating < 4 were deleted from the table on the `dev` branch.` to be after the first image to give some separation between them to make it more obvious that they're not just the same image